src/target/stm32f1.c

```diff
@@ -418,13 +418,17 @@ static bool stm32f1_option_erase(target *t)
 	return true;
 }
 
-static bool stm32f1_option_write_erased(target *t, uint32_t addr, uint16_t value)
+static bool stm32f1_option_write_erased(target *t, uint32_t addr, uint16_t value, bool width_word)
 {
 	if (value == 0xffff)
 		return true;
 	/* Erase option bytes instruction */
 	target_mem_write32(t, FLASH_CR, FLASH_CR_OPTPG | FLASH_CR_OPTWRE);
-	target_mem_write16(t, addr, value);
+	if (width_word) {
```

Comment:
the {}'s in this are redundant and not part of the code style. Please remove them